examples/list_hosts.py

```diff
@@ -20,57 +27,43 @@ def usage():
     print '-j|--json: Print output as json'
     print 'You can find your token at https://app.sysdigcloud.com/#/settings/user'
     sys.exit(1)
-
 try:
     opts, args = getopt.getopt(sys.argv[1:],"jd:",["json", "duration="])
 except getopt.GetoptError:
     usage()
-
-duration = 600
+duration = 3600
 print_json = False
 for opt, arg in opts:
     if opt in ("-d", "--duration"):
         duration = int(arg)
     elif opt in ("-j", "--json"):
         print_json = True
-
-if len(args) != 1:
-    usage()
-
 sdc_token = args[0]
-
-#
-# Instantiate the SDC client
-#
-sdclient = SdcClient(sdc_token)
-
 #
-# Prepare the query's metrics list. In this case, we have just one metric:
-# host.hostName. This is a 'key' metric, and we don't include any number metric.
-# Essentially, we create an 'enumeration' of hostnames.
 #
-metrics = [{"id": "host.hostName"}]
-
-#
-# Fire the query.
-# Note: there's no sampling time. This means that we're requesting the result to
-#       come as a single sample.
-#
-res = sdclient.get_data(metrics, # metrics list
-                        -duration,   # cover the last duration seconds...
-                        0,      # ... ending now...
-                        duration)    # ... with just one durations sample
-
-#
-# Show the results!
+# Prepare the query's metrics list.
+# In this case, we have one tag (used for segmentation) and one metric::
+# host.hostName. This is a tag, to identify each item of the output.
+# container.count: This is the metric
 #
+# Instantiate the SDC client
+sdclient = SdcClient(sdc_token)
+metrics = [{"id": "host.hostName"}, {"id": "container.count","aggregations":{"time":"avg","group":"avg"}}]
+res = sdclient.get_data(
+    metrics, -duration, 0, duration, paging={
+        "from": 0,
+        "to": 999
+    })
 if res[0]:
-    data = res[1]
+    output = []
+    data = res[1]['data']
+    for i in range(0, len(data)):
+        sample = data[i]
+        metrics = sample['d']
+        hostName = metrics[0]
+        count = metrics[1]
+        output.append('%s\t%d' % (hostName, count))
+    print '\n'.join(output)
```

Comment:
We can maybe add here a one liner, would this work?:
```python
print('\n'.join(["%s\t%s" % (data['d'][0], data['d'][1]) for data in res[1]['data']])
```

Reply:
It would definitively work.

However, I would avoid patterns that could be a little cryptic, especially in the context of these examples that serve the purpose of documenting what can be done and how. For instance, `hostName = metrics[0]` would be unnecessary in most cases, here the goal was to explain that `metrics` is an array, and the first element is the `hostName` value related to the first metric specified in the list.

Hope this makes sense :-)